styles/style.css

```diff
@@ -0,0 +1,246 @@
+* {
+    margin:0;
+    padding: 0;
+}
+
+.centered {
+    display: flex;
+    justify-content: center;
+}
+
+body {
+    font-family: sans-serif;
+    font-size: 4em;
+}
+
+.parallax_wrapper {
+    height:100vh;
+    overflow-x: hidden;
+    overflow-y: auto;
+    perspective: 300px;
+}
+
+.intro_outro_screen {
+    height: 100vh;
+    align-items: center;
+
+}
+
+.parallax_group {
+    position: relative;
+    height: 100vh;
+    transform-style: preserve-3d;
```

Comment:
Not gonna lie, I have no idea what this does, even after reading the description. But this isn't unusual for CSS. There are many style rules that we tend to encounter through a tutorial or inspecting an example we like. It's a good idea to then pare down the sample to the minimum that we need to maintain the effect, and see what different values for the rules change. This can be a great way to build CSS skills if you find it interesting.